src/main/host/syscall/handler/mod.rs

```diff
@@ -70,6 +70,18 @@ impl SyscallHandler {
             }};
         }
 
+        macro_rules! unsupported {
+            ($name: literal) => {{
+                log::warn!(
```

Comment:
Should we make this `warn_once_then_debug`?

I'm not sure but I think maybe yes because getting to this code path is not a syscall routing programming mistake, but rather could result from valid application usage calling syscalls that we don't yet support (but the app might have a valid fallback path). (By comparison, I think the `shim_only!` macro is a syscall routing programming mistake so it makes sense to panic in that case.)

Reply:
Changed this to `warn_once_then_debug`.